src/rgw/rgw_admin_common.h

```diff
@@ -0,0 +1,291 @@
+#ifndef CEPH_RGW_ADMIN_COMMON_H
+#define CEPH_RGW_ADMIN_COMMON_H
+
+#include "cls/rgw/cls_rgw_types.h"
+
+#include "common/ceph_json.h"
+#include <common/safe_io.h>
+
+#include "rgw_common.h"
+#include "rgw_rados.h"
+
+#define dout_context g_ceph_context
+#define dout_subsys ceph_subsys_rgw
+
+enum RgwAdminCommandGroup {
```

Comment:
please organize them methodically either by alphabetical order or by the order they are in RgwAdminCommand

Reply:
Fixed.